File: wallendar-app/app/src/main/java/com/td/wallendar/groupbalance/ui/GroupBalanceView.java
package com.td.wallendar.groupbalance.ui;

import com.td.wallendar.models.Debt;

import java.util.List;

public interface GroupBalanceView {
    void bind(List<Debt> debts);

    void failedToLoadDebts();

    void failedToSettleUpDebt();

    void onSettleUpPaymentDone(Debt debt);
}

File: wallendar-app/app/src/main/java/com/td/wallendar/groupbalance/ui/GroupBalanceView.kt
package com.td.wallendar.groupbalance.ui

import com.td.wallendar.models.Debt

interface GroupBalanceView {
    fun bind(debts: MutableList<Debt>)
    fun failedToLoadDebts()
    fun failedToSettleUpDebt()
    fun onSettleUpPaymentDone(debt: Debt)
}